AvalonStudio/AvalonStudio/Controls/Editor/EditorViewModel.cs

```diff
@@ -110,6 +122,26 @@ public override void OnClose()
             disposables.Dispose();
         }
 
+        public void OnPointerWheelChanged(PointerWheelEventArgs e)
```

Comment:
Does this guarantee that mouse wheel deltas are rounded to 10% values. So we don't ever get a problem not being able to wheel back to 100%?